### PR TITLE
Completed css-positioning exercise. | Parker Siu

### PR DESCRIPTION
**Pull request by parkersiu**
Screen Shot 2022-10-14 at 9 22 27 AM
Screen Shot 2022-10-14 at 9 50 01 AM
Kapture 2022-10-14 at 09 47 36

### CODE REVIEW

File: css-positioning/relative.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+.pokemon > img {
+  width: 150px;
+
+  /* your code here */
+}
+
+#bulbasaur {
+  /* your code here */
```
Comment (Zhiwutian) [inline css-positioning/relative.css:14]: don't forget to move poor Bulbasaur a little!

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
tiny update!

File: css-positioning/relative.css (hunk 1 above)
Comment (parkersiu) [reply css-positioning/relative.css]: Screen Shot 2022-10-14 at 9 54 00 AM
Poor Bulbasaur has been moved.

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**